soc/arm/nxp_lpc/lpc55xxx/soc.h

```diff
@@ -32,6 +32,8 @@
 #define IOCON_PIO_FUNC7 0x07u         /*!<@brief Selects pin function 7 */
 #define IOCON_PIO_FUNC9 0x09u         /*!<@brief Selects pin function 9 */
 #define IOCON_PIO_FUNC10 0x0Au        /*!<@brief Selects pin function 10 */
+#define IOCON_PIO_FUNC11 0x0Bu        /*!<@brief Selects pin function 11 */
```

Comment:
This change should either be in a separate commit, or the commit message needs to be updated to reflect that you are defining the missing PIO_FUNC11 definition.

Reply:
Again, regarding the commit here: Can you make the commit message slightly more verbose? "Added MCUX_PWM dependency; missing funct def; enabled submodule clocks" leaves me wondering what function definition was missing, it is not clear you are talking about a pin function versus a C function until you look at the commit diff.